docs/integrations/sources/opuswatch.md

```diff
@@ -0,0 +1,38 @@
+# OPUSWatch
+Connect to your OPUS data
```

Comment:
Can you provide a better description of what OPUS is? What is the website and API documentation?